MovingBlocks/Terasology
# chore[build]: convert facades/PC to gradle to kts

This converts the gradle config from groovy to kotlin.

This PR is intended to be a near line-for-line port. Hold any further changes for another branch that won't be so cluttered by the conversion.

blocking #4157, as it needs to make changes to these JavaExec tasks.

### How to test

Run `gradlew :facades:PC:tasks` for the list of tasks.

* test  the **Terasology run tasks**
  * See [Playing#Configuring a Server](docs/Playing.md#configuring-a-server) for information about how to configure the server settings from the gradle command line.
* test the **Terasology dist tasks**
  * dists should include jar files for all the modules you have sources checked out for in `/modules/`
